.github/workflows/check-dist.yml

```diff
@@ -4,6 +4,9 @@
 # For our project, we generate this file through a build process from other source files.
 # We need to make sure the checked-in `index.js` actually matches what we expect it to be.
 name: Check dist/
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
```suggestion
```

I find it strange that it needs a write-permission on `pull-requests` when all it does is `upload-artifacts`. 
Can we try without first?

Reply:
Actually the repo says that it needs write permissions:
```
This action requires the write permission for the [permissions.pull-requests scope](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idpermissions).
```
https://github.com/github/webpack-bundlesize-compare-action